searx/templates/simple/macros.html

```diff
@@ -21,9 +21,19 @@
 {% macro result_header(result, favicons, image_proxify) -%}
 <article class="result {% if result['template'] %}result-{{ result.template|replace('.html', '') }}{% else %}result-default{% endif %} {% if result['category'] %}category-{{ result['category'] }}{% endif %}{% for e in result.engines %} {{ e }}{% endfor %}">
   {{- result_open_link(result.url, "url_wrapper") -}}
+  {% if not rtl %}
+    {%- if favicon_resolver != "" %}
+    <div class="favicon"><img loading="lazy" src="{{ favicon_url(result.parsed_url.netloc) }}"></div>
```

Comment:
favicons are now loaded lazy (will save some bandwidth)

Reply:
Looks good - I was never able to get it working, but do you think we can avoid the rtl formatting in the html?

Reply:
I have not yet found a solution (TBH I have not yet dealt it) .. @Bnyro suggests CSS in https://github.com/searxng/searxng/pull/3727#pullrequestreview-2240912152 .. I'm not sure how.